Signal/src/ViewControllers/AppSettings/OWSLinkedDevicesTableViewController.m

```diff
@@ -44,7 +44,6 @@ - (void)viewDidLoad
 
     self.isExpectingMoreDevices = NO;
     self.tableView.rowHeight = UITableViewAutomaticDimension;
-    self.tableView.estimatedRowHeight = 70;
```

Comment:
Why are you removing the estimatedRowHeights here and elsewhere?

Reply:
We could go either way here.

`estimatedRowHeight` improves perf for tables with many cells.  But I'm not sure that we're supplying a good estimate since our cell heights can vary wildly based on dynamic text size.  The docs don't make clear to me if an inaccurate estimate is better or worse for perf than no estimate at all.  WDYT?

Reply:
I would leave it as is. It's true we don't know the inner workings of estimatedRowHeight, but based on the documentation, we should be providing a reasonable guess, which we were.

Reply:
I've already put it back in another PR.  You were right!